rpc/types.go

```diff
@@ -106,6 +108,9 @@ func (bn *BlockNumber) UnmarshalJSON(data []byte) error {
 	case "pending":
 		*bn = PendingBlockNumber
 		return nil
+	case "finalized":
```

Comment:
Looks like you forgot to add it to `func (bnh *BlockNumberOrHash) UnmarshalJSON(data []byte) error {`

Reply:
Yes, its added now. Thanks!